services/ListingService.ts

```
@@ -78,6 +78,128 @@ export class ListingService {
+            limit,
+            totalPages: 0
+          }
+        };
+      }
+
+      // Extract listing IDs from keys
+      const listingIds = listingKeys.map(key => key.split(':').pop() as string);
+
+      // Get all listings by their IDs
+      let listings = await Promise.all(
+        listingIds.map(id => this.getListingById(siteId, id))
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)